.github/workflows/package-component.yaml

```diff
@@ -40,18 +40,29 @@ jobs:
         with:
           go-version-file: smithyctl/go.mod
 
-      - name: Build CLI
-        run: make smithyctl/cmd/bin
-
       - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run Packaging
+      - name: Build CLI, component images and package
         run: |
+          set -ex
+          make smithyctl/bin
+          ./bin/smithyctl/cmd/linux/amd64/smithyctl \
+            --debug-enabled=true \
+            --debug-level=debug \
+            component \
+              build \
+                --registry-auth-enabled=true \
+                --registry-auth-username=${{ github.actor }} \
+                --registry-auth-password=${{ secrets.GITHUB_TOKEN }} \
+                --registry-url=ghcr.io \
+                --tag=${{ steps.tag_info.outputs.version }} \
```

Comment:
let's also add the new sdk-version flag here

Reply:
fixed